source/atomics.tex

```diff
@@ -3701,3 +3701,140 @@
+using std::@\libglobal{atomic_uintmax_t}@            // \seebelow
+
+using std::@\libglobal{atomic_is_lock_free}@                         // \seebelow
+using std::@\libglobal{atomic_load}@                                 // \seebelow
+using std::@\libglobal{atomic_load_explicit}@                        // \seebelow
+using std::@\libglobal{atomic_store}@                                // \seebelow
+using std::@\libglobal{atomic_store_explicit}@                       // \seebelow
+using std::@\libglobal{atomic_exchange}@                             // \seebelow
+using std::@\libglobal{atomic_exchange_explicit}@                    // \seebelow
+using std::@\libglobal{atomic_compare_exchange_strong}@              // \seebelow
+using std::@\libglobal{atomic_compare_exchange_strong_explicit}@     // \seebelow
+using std::@\libglobal{atomic_compare_exchange_weak}@                // \seebelow
+using std::@\libglobal{atomic_compare_exchange_weak_explicit}@       // \seebelow
+using std::@\libglobal{atomic_fetch_add}@                            // \seebelow
+using std::@\libglobal{atomic_fetch_add_explicit}@                   // \seebelow
+using std::@\libglobal{atomic_fetch_sub}@                            // \seebelow
+using std::@\libglobal{atomic_fetch_sub_explicit}@                   // \seebelow
+using std::@\libglobal{atomic_fetch_or}@                             // \seebelow
+using std::@\libglobal{atomic_fetch_or_explicit}@                    // \seebelow
+using std::@\libglobal{atomic_fetch_and}@                            // \seebelow
+using std::@\libglobal{atomic_fetch_and_explicit}@                   // \seebelow
+using std::@\libglobal{atomic_flag_test_and_set}@                    // \seebelow
+using std::@\libglobal{atomic_flag_test_and_set_explicit}@           // \seebelow
+using std::@\libglobal{atomic_flag_clear}@                           // \seebelow
+using std::@\libglobal{atomic_flag_clear_explicit}@                  // \seebelow
+
+using std::@\libglobal{atomic_thread_fence}@                         // \seebelow
+using std::@\libglobal{atomic_signal_fence}@                         // \seebelow
+\end{codeblock}
+
+\pnum
+Each \grammarterm{using-declaration} for some name $A$ in the synopsis above
+makes available the same entity as \tcode{std::$A$}
+declared in \libheaderrefx{atomic}{atomics.syn}.
+Each macro listed above other than \tcode{_Atomic(T)}
+is defined as in \libheader{atomic}.
+It is unspecified whether \libheader{stdatomic.h} makes available
+any declarations in namespace \tcode{std}.
+
+\pnum
+Each of the \grammarterm{using-declaration}s for
+\tcode{int$N$_t}, \tcode{uint$N$_t}, \tcode{intptr_t}, and \tcode{uintptr_t}
+listed above is defined if and only if the implementation defines
+the corresponding typedef name in \ref{atomics.syn}.
+
+\pnum
+Neither the \tcode{_Atomic} macro,
+nor any of the non-macro global namespace declarations,
+are provided by any C++ standard library header
```

Comment:
"C++" -> "\Cpp{}"

Reply:
Looks like a tool check might be needed.

Reply:
There are a lot of `extern "C++"` things floating around...

Reply:
... fixed